### ISSUE TITLE
Server debug

### ISSUE THREAD

**Issue by katyhuff** [scipy-conference/procbuild#4]
@sarostru and @sbenthall : it would be best if one of you could serve procbuild on zibi from your user account (this way, you can respond to authors needs directly without me in the way). Since people are starting to submit, the server needs to be up and running at all times, to allow submitting authors to see whether their pr is building. 

I have it serving right now from my user account on zibi but something is funky (http://zibi.bids.berkeley.edu:5000/) . While it finds the paper PR (through the github webhook), doesn't seem to be trying to build it.  ... anyone feel like debugging? @stefanv I don't remember what it means when the brackets are empty... 

This is how it went down:

```
khuff@maria:~ $ ssh huff@zibi.bids.berkeley.edu
Last login: Wed May 25 11:49:50 2016 from 169.229.224.158
zibi:~ huff$ source activate py27
discarding /Users/huff/anaconda3/bin from PATH
prepending /Users/huff/anaconda3/envs/py27/bin to PATH
(py27)zibi:~ huff$ cd procbuild/
(py27)zibi:procbuild huff$ ./update_prs
Fetching page 1 (state=open) from scipy-conference/scipy_proceedings...
/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/urllib3/connectionpool.py:790: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)
Fetching page 2 (state=open) from scipy-conference/scipy_proceedings...
/Users/huff/anaconda3/envs/py27/lib/python2.7/site-packages/urllib3/connectionpool.py:790: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)
(py27)zibi:procbuild huff$ python runserver.py &
[1] 98515
(py27)zibi:procbuild huff$ Setting up build queue...
Launching queue monitoring process...
serving on http://0.0.0.0:5000
```

**Comment by katyhuff** [comment]
nvm for now, I'm tracking it down. Had the wrong branch name in the server script.. That seems to be half the problem.

**Comment by stefanv** [comment]
Run `build_papers.py` to get a first set of PDFs.  Thereafter, changes to
the paper should trigger builds.

**Comment by katyhuff** [comment]
It's actually more problematic than that. Have to change use of paper_queue.qsize() because it's unsupported on mac. PR on its way.

**Comment by katyhuff** [comment]
see #5

**Comment by katyhuff** [comment]
ok, now we're getting somewhere... zibi has only the most basic latex on it so we're getting more reasonable errors now:



@stefanv ... shall I go ahead and install MacTeX on zibi or will that screw stuff up for you somehow?

**Comment by stefanv** [comment]
I had no luck trying that recently; let me try again.

**Comment by stefanv** [comment]
MacTeX now installed on zibi

**Comment by katyhuff** [comment]
Thanks!


On Wed, May 25, 2016 at 2:08 PM, Stefan van der Walt <
notifications@github.com> wrote:

> MacTeX now installed on zibi
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/scipy-conference/procbuild/issues/4#issuecomment-221708118>
>



-- 
http://katyhuff.github.com

**Comment by katyhuff** [comment]
success y'all.

On Wed, May 25, 2016 at 2:10 PM, Katy Huff <katyhuff@gmail.com> wrote:

> Thanks!
>
>
> On Wed, May 25, 2016 at 2:08 PM, Stefan van der Walt <
> notifications@github.com> wrote:
>
>> MacTeX now installed on zibi
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/scipy-conference/procbuild/issues/4#issuecomment-221708118>
>>
>
>
>
> --
> http://katyhuff.github.com
>



-- 
http://katyhuff.github.com
